src/status_im/translations/fr.cljs

```diff
@@ -4,26 +4,29 @@
   {
    ;common
    :members-title                         "Membres"
-   :not-implemented                       "! Pas disponible"
-   :chat-name                             "Pseudo"
+   :not-implemented                       "! Non disponible"
+   :chat-name                             "Pseudonyme"
    :notifications-title                   "Notifications et sons"
    :offline                               "Hors ligne"
    :search-for                            "Rechercher..."
    :cancel                                "Annuler"
    :next                                  "Suivant"
+   :open                                  "Ouvrir"
+   :description							  "Description"
```

Comment:
can you align the translations with the others ?

Reply:
sure, fixing it.